### ISSUE TITLE
Caret loosing alignment when within scrollable container

### ISSUE THREAD

**Issue by gronke** [PolymerElements/paper-input#156]
When used within a container with `{ overflow: scroll; }` the caret looses its alignment when scrolling vertically on Mobile Safari (iOS 8.4.1)

### Demo
![polymer-paper-element-scrolling-bug](https://cloud.githubusercontent.com/assets/473924/9425604/0cb8252a-4916-11e5-9e0f-dc0e5c68c1dd.gif)

### Source
```
<html>
  <head>
    <script src="bower_components/webcomponentsjs/webcomponents.js"></script>
    <meta name="viewport" content="width=200">
    <style>
  
      body,
      html {
        position: fixed;
      }

      .wrapper {
        position: fixed;
        top: 0px;
        left: 0px;
        right: 0px;
        bottom: 0px;
        border: 1px solid red;
        overflow: scroll;
      }

      .spacer {
        width: 100%; 
        background: orange
      }

    </style>
  </head>
  <body>
    <link rel="import" href="bower_components/paper-input/paper-input.html">

    <div class="wrapper">
      <div class="spacer" style="height: 150px;"></div>
      <h1>Label</h1>
      <paper-input label="Demo Label"></paper-input>
      <div class="spacer" style="height: 600px;"></div>
    </div>

    <script>
      document.addEventListener('touchmove', function(e) {
        e.stopPropagation();
      });
    </script>
  </body>
</html>
```

**Comment by notwaldorf** [comment]
I cannot repro this on iOS9 (jsbin i used: http://output.jsbin.com/yisuqe). Please re-open if this still occurs

**Comment by gronke** [comment]
@notwaldorf thanks for looking into this - it's fixed in iOS 9.1
